saleor/graphql/product/tests/test_bulk_delete.py

```diff
@@ -249,6 +249,34 @@ def test_delete_products(
     assert OrderLine.objects.filter(pk__in=not_draft_order_lines_pks).exists()
 
 
+@patch("saleor.plugins.webhook.plugin.trigger_webhooks_for_event.delay")
+def test_delete_products_trigger_webhook(
+    mocked_webhook_trigger,
+    staff_api_client,
+    product_list,
+    permission_manage_products,
+    channel_USD,
+    settings,
+):
+    # given
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
+
+    query = DELETE_PRODUCTS_MUTATION
+    variables = {
+        "ids": [
+            graphene.Node.to_global_id("Product", product.id)
+            for product in product_list
+        ]
+    }
+    response = staff_api_client.post_graphql(
+        query, variables, permissions=[permission_manage_products]
+    )
+    content = get_graphql_content(response)
+
+    assert content["data"]["productBulkDelete"]["count"] == 3
+    assert mocked_webhook_trigger.called
```

Comment:
Maybe `assert_called_once_with` ?

Reply:
This webhook is called three times in this mutation. 
I already tested with params are called when this webhook is triggered in `/saleor/plugins/webhook/tests/test_webhook.py` `test_product_deleted`.